app/controllers/invite_controller.rb

```diff
@@ -81,17 +82,15 @@ def submit
         @message = t(:message_email_exists)
         @type = "danger"
       else
-        tester = Spaceship::Tunes::Tester::External.new({
-          'emailAddress' => {'value' => email},
-          'firstName' => {'value' => first_name},
-          'lastName' => {'value' => last_name}
-        })
+        tester = Spaceship::Tunes::Tester::External.create!(email: email,
+                                                       first_name: first_name,
+                                                        last_name: last_name)
 
         logger.info "Successfully created tester #{tester.email}"
 
         if apple_id.length > 0
           logger.info "Addding tester to application"
-          tester.add_to_app!(apple_id)
```

Comment:
This code is a bit subject to errors because after TestFlight's changes, there only _could be_ a group that is flagged as default. There is a PR here https://github.com/fastlane/fastlane/pull/8943 that handles this case for _pilot_ to add testers go other groups specified in _pilot_'s options. I think that we may need to add something to _boarding_ that allows the user to set the group(s) to which they want testers signing up via _boarding_ to be added.